text/off-chain-workers/indexing.md

```diff
@@ -0,0 +1,60 @@
+# Off-chain Indexing
+
+There are times when on-chain extrinsics need to pass data to the off-chain worker context, such as pass the extrinsic caller account to off-chain worker for further computation.
+
+We can surely pass this piece of data via on-chain storage, but this is costly and it will make the data propagate among the blockchain network. If this is not a piece of critical information that need to have consensus upon the whole network, another way is to pass this data via off-chain indexing.
+
+We will demonstrate this in [`ocw-demo` pallet](https://github.com/substrate-developer-hub/recipes/tree/master/pallets/ocw-demo/src/lib.rs).
+Knowledge discussed in this chapter built upon [using local storage in off-chain worker context](./storage.md).
+
+> **Notes**
+>
+> In order to see the off-chain indexing feature in effect, please run the kitchen node with
+> off-chain indexing flag on, as `./target/debug/kitchen-node --dev --tmp --enable-offchain-indexing true`
```

Comment:
You've used the debug path here. Is that intentional? I just realized the recipes no longer explains how to run a node.